ember-toucan-core/src/components/form/controls/textarea.ts

```diff
@@ -0,0 +1,18 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export interface ToucanFormTextareaControlComponentSignature {
+  Element: HTMLTextAreaElement;
+  Args: {
+    isDisabled?: boolean;
+    onChange?: (value: string, e: Event | InputEvent) => void;
```

Comment:
I updated this a little bit based on https://github.com/CrowdStrike/ember-toucan-core/pull/36/files#r1114409957. I personally like having access directly to the event when needed, so adding it as a second argument felt like a potential nice thing to have.  Some existing code is doing this as well, FWIW.  For example, one case was not just wanting the `value` but the `key`/`keyCode` - I could access it directly from `e`.  Totally open to suggestions here though as well!